apps/view/src/render/files.ts

```
@@ -77,12 +77,12 @@ function fileReader(file: File): FileStream {
 
 async function zipReader(file: Blob): PromiseArray<FileStream> {
   return import('jszip')
-    .then(ZipModule => ZipModule.loadAsync(file))
-    .then(zip =>
-      Object.keys(zip.files)
-        .filter(name => !zip.files[name].dir)
-        .map(name =>
-          pump<FileStream>(zip.files[name].nodeStream(), new FileStream(name))
+        .then(ZipModule => ZipModule.loadAsync(file))
```

ESLint

[error] Delete `····`

(prettier/prettier)